### DISCUSSION TITLE
Nesquik game freeze after Level 4, how can I search in any log?

### DISCUSSION THREAD

**Discussion by littbarski** [joncampbell123/dosbox-x#5290] [General]
hello, 

we play many old games like Kelloggs, Eismann and now Nesquik 
([Tricky Quiky Games 2](https://werbespiel.blogspot.com/2010/07/tricky-quiky-games-2.html)). 
As said in the title, how can I activate a log of DosBox-X to search for the error?

the problem is, that DosBox-X is closed directly when the error appears, so the window of DosBox-X and the Debugger is not to use anymore.

(perhaps somebody could play the game already, but I am also in general asking what to do when DoxBox-X closes directly, how to find a solution then.

**Comment by joncampbell123** [comment]
Add this to your dosbox.conf:

    [log]
    logfile=log.txt

Then, run dosbox-x with the --debug option.

**Comment by littbarski** [comment]
hello and thank you very much!
First time I opened a sav file perhaps too early I got this line (very) often:
**76475239 WARN  DMA_CONTROL:BUG: Attempted DMA channel read when DMA channel is configured by guest for writing (to memory)**

Also I get a warning like this always:
**13 WARN  MISC:MEM_SlowPath called within system RAM at page c4**

But what I in general find as a first (for me as a non developer) issue could be:
[...]
95638720       PIT:PIT 0 Timer at 18.2065 Hz mode 2
  95655843       INT10:Set Video Mode 13
  95655843       VGA:Blinking 0
  95655843       VGA:Blinking 8
  95655843       MOUSE:New video mode is the same as the old
  **95660425 DEBUG VGAMISC:Render On Demand mode is off for RodU -1**
  95660425       VGA:h total 100 end  80 blank ( 80/ 98) retrace ( 84/ 96)
  95660425       VGA:v total 449 end 400 blank (407/442) retrace (412/414)
  95660425       VGA:VGA forced refresh rate in effect, 60.000
  95660425       VGA:screen: 1.333, scanfield: 1.312, scan: 1.016, vratio: 0.800
  95660425       VGA:h total 0.03712 (26.94kHz) blank(0.02970/0.03638) retrace(0.03118/0.03563)
  95660425       VGA:v total 16.66667 (60.00Hz) blank(15.10765/16.40683) retrace(15.29324/15.36748)
  95660425       VGA:video clock: 25.18MHz mode M_VGA
  95830789       PIT:PIT 0 Timer at 18.2068 Hz mode 2
[...]
[...]
 121157541       PIT:PIT 0 Timer at 18.2065 Hz mode 2
**121650673 DEBUG DOSMISC:DOS_FreeMemory(seg=0x0f02)**
 121658460 DEBUG DOSMISC:Flush STDIN
 121658976 DEBUG MISC:Silencing PC speaker output (output disabled)
[...]

and later then
121666948 DEBUG MISC:Emulation threw DOSBox-X kill switch signal
121666948 DEBUG MISC:Shutting down GFX renderer
121666948 DEBUG CPU:Callback dump


My guess is that this is a memory issue, which settings could a user try in such a case in the DOSBox-X conf file? 
Or any other hint like LOADFIX (?).

**Comment by littbarski** [comment]
Unfortunately I still have the same error. As I am not a developer, I would just like to ask, which DOS commands I could try to test different memory settings? So I would like to leave the conf as it is, start DOSBox-X and then try from there some commands (each time new boot).
